src/components/Body.js

```diff
@@ -1,15 +1,23 @@
 import React from 'react'
 import { InstantSearch } from 'react-instantsearch-dom'
+import styled from 'styled-components'
 
 import { useMeilisearchClientContext } from 'context/MeilisearchClientContext'
 import Box from 'components/Box'
 import Header from 'components/Header/index'
+import RightPanel from 'components/RightPanel'
 import BodyWrapper from 'components/BodyWrapper'
 import EmptyView from 'components/EmptyView'
 import OnBoarding from 'components/OnBoarding'
 import Results from 'components/Results'
 import Typography from 'components/Typography'
 
+const ContentWrapper = styled.div`
+  width: ${({ isRightPanelOpen }) =>
+    isRightPanelOpen ? 'calc(100% - 430px)' : '100%'};
```

Comment:
Can you create a CSS variable for the width of the side panel? 
We didn't for the header and the banner and it was a mess to maintain 🤣 Having one for the side panel would make it much easier to maintain if we want to change it some day :)

Reply:
Added `sizes.rightPanel` to the theme ☑️